lib/Conversion/ATenToXTenPass.cpp

```diff
@@ -145,6 +144,53 @@ bool isLongestBranch(OpResult left, OpResult right) {
   return true;
 }
 
+// Tablegen adapter to check if attributes allow a conversion from `ReduceMean`
+// to `GlobalAveragePool`.
+bool isReduceMeanGlobalAveragePool2D(Value dims, Value keepdims) {
+
+  // Get defining operands
```

Comment:
```suggestion
  // Get defining operands
```
The comment doesn't add any information.